Content/NPCs/Town/BlacksmithNPC.cs

```diff
@@ -41,21 +42,26 @@ public override void SetDefaults()
 		NPC.knockBackResist = 0.4f;
 		AnimationType = NPCID.Guide;
 		
-		NPC.TryEnableComponent<NPCDeathEffects>(
+		NPC.TryEnableComponent<NPCHitEffects>(
 			c =>
 			{
-				c.AddGore($"{PoTMod.ModName}/{Name}_0", 1);
-				c.AddGore($"{PoTMod.ModName}/{Name}_1", 2);
-				c.AddGore($"{PoTMod.ModName}/{Name}_2", 2);
+				c.AddGore(new NPCHitEffects.GoreSpawnParameters($"{PoTMod.ModName}/{Name}_0", 1, static npc => npc.life <= 0));
+				c.AddGore(new NPCHitEffects.GoreSpawnParameters($"{PoTMod.ModName}/{Name}_1", 1, static npc => npc.life <= 0));
+				c.AddGore(new NPCHitEffects.GoreSpawnParameters($"{PoTMod.ModName}/{Name}_2", 2, static npc => npc.life <= 0));
```

Comment:
Thoughts on making a method that is equal to all these delegates? Having to repeat it every time is a little odd. Inconsequential, but awkward.

Reply:
We should probably keep those somewhere since they are used universally by other components as well.